### PR TITLE
Add SDL2_gfx port

### PR DESCRIPTION
**Pull request by zekoz**

### CODE REVIEW

**Comment by kripken** [comment]
Looks good @zekoz, thanks, just needs to merge or rebase for current incoming.

File: embuilder.py [hunk 1]
```diff
@@ -101,8 +101,8 @@
         }
       '''
 
-SYSTEM_TASKS = ['compiler-rt', 'libc', 'libc-mt', 'libc-extras', 'emmalloc', 'emmalloc_debug', 'dlmalloc', 'dlmalloc_threadsafe', 'pthreads', 'dlmalloc_debug', 'dlmalloc_threadsafe_debug', 'libcxx', 'libcxx_noexcept', 'libcxxabi', 'html5']
-USER_TASKS = ['al', 'gl', 'binaryen', 'bullet', 'freetype', 'icu', 'libpng', 'ogg', 'sdl2', 'sdl2-image', 'sdl2-ttf', 'sdl2-net', 'vorbis', 'zlib']
+SYSTEM_TASKS = ['compiler-rt', 'libc', 'libc-mt', 'libc-extras', 'emmalloc', 'emmalloc_debug', 'dlmalloc', 'dlmalloc_threadsafe', 'pthreads', 'dlmalloc_debug', 'dlmalloc_threadsafe_debug', 'libcxx', 'libcxx_noexcept', 'libcxxabi', 'html5']`
```
Comment (kripken) [inline embuilder.py]: Looks like there is an extra character on this line at the very end, a quote or a backtick, I think it's what caused the test failure.

Comment (zekoz) [reply embuilder.py]: Serves me right for using github.com's merge tool. Pushed a proper rebase.

**Comment by kripken** [comment]
Great, thanks!

**Pull request merged**